.github/workflows/maven.yaml

```diff
@@ -0,0 +1,29 @@
+name: maven
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    name: Build
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2022, macos-12]
+        java: [8, 11]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+      - uses: actions/cache@v3
```

Comment:
No need, see https://github.com/actions/setup-java

Reply:
@embano1 should I just delete the actions/cache@v3 action from the `maven.yaml`?